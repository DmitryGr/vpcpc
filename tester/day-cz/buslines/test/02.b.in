100
64 42
6 10
7 13
18 16
64 38
69 79
48 35
14 82
34 98
41 12
93 15
90 26
32 42
77 65
21 3
68 48
34 33
83 8
2 95
4 43
10 58
51 20
56 96
100 81
24 43
81 62
54 28
45 74
63 76
98 57
29 66
76 94
49 74
18 55
15 19
47 55
7 21
73 31
37 25
45 16
50 38
47 32
57 59
61 14
99 44
39 33
84 35
94 27
11 75
71 52
23 63
4 41
70 72
30 56
6 36
2 25
28 87
99 85
77 22
17 90
65 24
13 70
26 46
88 72
40 61
78 12
39 73
82 51
53 92
22 67
68 97
67 53
37 79
17 5
97 8
40 49
9 91
80 75
92 29
58 50
86 83
91 1
46 36
85 84
31 11
60 5
89 62
23 19
66 88
78 86
27 80
52 69
44 30
100 96
9 93
71 20
54 95
3 60
59 89
