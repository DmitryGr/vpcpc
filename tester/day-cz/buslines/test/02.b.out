99
479
2439
2507
2393
2207
2477
2393
293
2169
1217
2477
2493
1289
479
1679
2367
1737
569
1067
2459
2547
657
2529
569
2309
989
197
2543
2087
1289
1899
1493
1557
2277
2243
657
2043
1427
1427
2493
1949
2519
2129
1619
2277
1847
2309
1493
2087
1143
909
2549
293
1793
2043
1679
2129
1737
2417
1359
1847
743
1997
2537
2537
2549
2339
827
2507
989
2519
1359
1557
1143
827
2543
2459
743
1067
1899
1217
2417
2243
2207
2439
99
2529
1793
2339
197
2547
387
909
387
1997
2367
1619
2169
1949
